src/id3/v2/tag.rs

```diff
@@ -872,10 +915,56 @@ impl<'a> Id3v2TagRef<'a, std::iter::Empty<FrameRef<'a>>> {
 
 // Create an iterator of FrameRef from a Tag's items for Id3v2TagRef::new
 pub(crate) fn tag_frames(tag: &Tag) -> impl Iterator<Item = FrameRef<'_>> + Clone {
+	fn create_frameref_for_number_pair<'a>(
+		number: Option<&str>,
+		total: Option<&str>,
+		id: &'a str,
+	) -> Option<FrameRef<'a>> {
+		let mut value: Option<Cow<'_, str>> = None;
+
+		if let Some(number) = number {
+			value = Some(Cow::Borrowed(number));
+		}
+
+		if let Some(total) = total {
+			let number = value.unwrap_or_else(|| Cow::Owned(DEFAULT_NUMBER.to_string()));
```

Comment:
A `const &str` representation for `DEFAULT_NUMBER` would avoid an unecessary allocation -> `value.as_deref().unwrap_or(DEFAULT_NUMBER_STR)`.

With the proposed changes and extension of `merge_num_pair` this might even become obsolete.

Reply:
I did not fixed it. Because `merge_num_pair` was changed.